person.rb

```diff
@@ -0,0 +1,23 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Randaom.rand(1..1000)
+    @age = age
+    @name = name
+    @parent_permission = parent_permission
+  end
+
+  # Private method to check age
+  private
+
+  def of_age?
+    true if @age >= 18
+  end
+
+  # Public method for if person is of age or if they have permission from parents
+  def can_use_services?
+    true if of_age? || @parent_permission
+  end
```

Comment:
Nice job so far😉. From the comments on your code, I am sure you understand that the `can_use_service?` method is meant to be a public method😉. But you seem to have made a mistake in your implementation. every method that comes after the `private` declaration is usually a private method. so you can either declare it as a public method or preferably place the method before the private declearation😉.

```suggestion
  # Public method for if person is of age or if they have permission from parents
  def can_use_services?
    true if of_age? || @parent_permission
  end

  # Private method to check age
  private

  def of_age?
    true if @age >= 18
  end
```